app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :users
```

Comment:
- [ ] Kindly refactor the comment to belong to an `author`, not `users`, and also refer to the **User** class.

- [ ] Kindly ensure the appropriate database schema column is renamed accordingly.